packages/playground/website/src/components/site-manager/site-manager-sidebar/index.tsx

```diff
@@ -40,65 +30,20 @@ export function SiteManagerSidebar({
 	siteSlug?: string;
 	onSiteClick: (siteSlug: string) => void;
 }) {
-	const [sites, setSites] = useState<Site[]>([]);
-
-	const generateSiteFromSlug = (slug: string): Site => {
-		let name = slug.replaceAll('-', ' ');
-		name = name.charAt(0).toUpperCase() + name.slice(1);
-		/**
-		 * Ensure WordPress is spelt correctly in the UI.
-		 */
-		name = name.replace(/wordpress/i, 'WordPress');
+	const sites = useSelector(
+		(state: PlaygroundReduxState) => state.siteListing.sites
```

Comment:
Let's also select loading/error status here and reflect in the UI, even if it is a crude or basic error message.